RecoLocalTracker/Phase2TrackerRecHits/plugins/Phase2StripCPEESProducer.cc

```diff
@@ -51,7 +51,7 @@ Phase2StripCPEESProducer::Phase2StripCPEESProducer(const edm::ParameterSet& p) {
 }
 
 std::unique_ptr<ClusterParameterEstimator<Phase2TrackerCluster1D> > Phase2StripCPEESProducer::produce(
-    const TkStripCPERecord& iRecord) {
+    const TkPhase2OTCPERecord& iRecord) {
```

Comment:
Could this. and the related modifications in ```RecoLocalTracker/Phase2TrackerRecHits/plugins/Phase2TrackerRecHits.cc``` be responsible of the differences observed in Phase2 outputs? (Just guesssing...)

Reply:
As the change below exactly produces the changes in the results we have observed before, I think this line is fine.